decidim-elections/config/locales/en.yml

```diff
@@ -259,6 +259,7 @@ en:
               'false': does not have a <strong>public key</strong>
               'true': has a <strong>public key</strong>
             requirements:
+              component_published: The election component is <strong>published</strong>.
```

Comment:
When its not published I see a translation missing at `en.decidim.elections.admin.steps.create_election.errors.component_published`:
![image](https://user-images.githubusercontent.com/717367/170942120-aaf3c1d1-59aa-4e14-81e9-0b11b1faa604.png)



Can you add this to line 249 🙏🏽? 
```yaml
               component_published: The election component is <strong>not published</strong>.
```

(GitHub doesn't allow me to make the suggestion at the line, as it's out of the scope of the review).